webapp/store/views.py

```diff
@@ -159,6 +159,29 @@ def details_overview(entity_name):
     # Remove Markdown comments
     readme = re.sub("(<!--.*-->)", "", readme, flags=re.DOTALL)
 
+    bases = package["default-release"]["revision"]["bases"]
+    channel_bases = []
+
+    for i in bases:
+        has_base = False
+
+        for b in channel_bases:
+            if b["name"] == i["name"]:
+                has_base = True
+
+        if not has_base:
+            channel_bases.append(
+                {
+                    "name": i["name"],
+                    "channels": [],
+                }
+            )
+
+    for i in channel_bases:
+        for b in bases:
+            if b["name"] == i["name"]:
+                i["channels"].append(b["channel"])
+
```

Comment:
I think this is great but we should do it on `logic.py`. Take a look into the `extract_series` function on logic.py I think we need to replace this function with another one that returns a similar output to this. And then include it on `add_store_front_data` so we can use something like `package.store_front.bases` on the templates everywhere.

If we remove `extract_series` we need to make sure is compatible with the previous code everywhere.